### PR TITLE
Move to 194 and clean up some unneeded dependencies

### PR DESCRIPTION
**Pull request by smillidge**
Signed-off-by: Steve Millidge <steve.millidge@payara.fish>

### CODE REVIEW

File: javaee/arquillian-example/pom.xml [hunk 1]
```diff
@@ -97,8 +96,8 @@
         </dependency>
         <dependency>
             <groupId>fish.payara.arquillian</groupId>
-            <artifactId>arquillian-payara-server-4-embedded</artifactId>
-            <version>1.0.Beta3</version>
+            <artifactId>arquillian-payara-server-embedded</artifactId>
+            <version>2.0</version>
```
Comment (MarkWareham) [inline javaee/arquillian-example/pom.xml]: Don't need to specify version, it's managed in the bom https://github.com/payara/Payara/blob/master/api/payara-bom/pom.xml

File: javaee/rest-request-jsf/pom.xml [hunk 2]
```diff
@@ -56,7 +56,7 @@
         <dependency>
             <groupId>com.sun.jersey</groupId>
             <artifactId>jersey-client</artifactId>
-            <version>1.19</version>
+            <version>1.19.4</version>
```
Comment (MarkWareham) [inline javaee/rest-request-jsf/pom.xml]: I believe the same here. Payara BOM utilises jersey bom

File: javaee/arquillian-example/pom.xml (hunk 1 above)
Comment (smillidge) [reply javaee/arquillian-example/pom.xml]: then the root parent pom of the examples needs further work to pull in the bom as an inherited dependency

Comment (MarkWareham) [reply javaee/arquillian-example/pom.xml]: In that case we'll consider it outside the scope of this change

Comment (MarkWareham) [reply javaee/arquillian-example/pom.xml]: Sorry, just noticed the project. This has been moved to payara/payara

**Pull request merged**